openwrt-openwisp-monitoring/files/netjson-monitoring.lua

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env lua
+-- retrieve monitoring information
+-- and return it as NetJSON Output
+local cjson = require('cjson')
+
+local ubus_lib = require('ubus')
+local ubus = ubus_lib.connect()
+if not ubus then
+    error('Failed to connect to ubusd')
+end
+
+local dhcp = require('dhcp')
+local interface_functions = require('interfaces')
```

Comment:
both variables, `interface` and `interface_functions` are defined :confused:

Reply:
How about `host_interface` for the variable identifier?